### PR TITLE
RDI-3208 remove: Rebase from the source branch

### PR DESCRIPTION
**Pull request by cfoch**
Issue: RDI_3208

### CODE REVIEW

**Comment by rgonzalezfluendo** [comment]
Adding a test for this issue would be nice.

**Comment by cfoch** [comment]
> Adding a test for this issue would be nice.

I struggle to create a passing unit test for it or my solution is not 100% correct.

**Comment by rgonzalezfluendo** [comment]
Adding more info in the commit msg with the corner case would be nice.

**Comment by rgonzalezfluendo** [comment]
CI is red

File: tests/test_remove.py [hunk 1]
```diff
@@ -65,3 +65,41 @@ def test_remove(self):
...
+            status = "integrated"
+
+            [[features]]
+            remote = "origin"
+            name = "example1-feature1-fixup"
+            pr = "https://github/fluendo/git-upstream-workflow/pull-requests/10"
+            status = "pending"
+        """
+        import guw
+        print("HOLA", guw)
```
Comment (rgonzalezfluendo) [inline tests/test_remove.py]: Hola

**Review by rgonzalezfluendo — CHANGES_REQUESTED** [review]
* Add a good commit msg
* Drop debug prints

**Review by rgonzalezfluendo — APPROVED** [review]
